src/documents/css/docs-template.css.styl

```diff
@@ -671,14 +671,14 @@ html body {
     li {
         padding: 0.6rem 0.2rem 0.6rem 0;
     }
-    
-/*! 
+
```

Comment:
Not sure why these lines are commented out. I will have to look into this later.